File: Visual Studio 2010/VBMultiThreadedWebDownloader/ErrorEventArgs.vb
'****************************** 模块头 ******************************'
' 模块名称:  ErrorEventArgs.vb
' 项目名称:	    VBMultiThreadedWebDownloader
' 版权 (c) Microsoft Corporation.
'  ErrorEventArgs 类 定义了由 ErrorOccurred 事件使用的参数。
'
' This source is subject to the Microsoft Public License.
' See http://www.microsoft.com/opensource/licenses.mspx#Ms-PL.
' All other rights reserved.
' 
' THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
' EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
' WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
'***************************************************************************'

Public Class ErrorEventArgs
    Inherits EventArgs
    Public Property Exception() As Exception
End Class

File: Visual Studio 2010/CSMultiThreadedWebDownloader/ErrorEventArgs.cs
/****************************** 模块头 ******************************\
* 模块名称:  ErrorEventArgs.cs
* 项目名称:	    CSMultiThreadedWebDownloader
* 版权 (c) Microsoft Corporation.
* 
*  ErrorEventArgs 类 定义了由 ErrorOccurred 事件使用的参数。
* 
* This source is subject to the Microsoft Public License.
* See http://www.microsoft.com/opensource/licenses.mspx#Ms-PL.
* All other rights reserved.
* 
* THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
* EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
* WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
\***************************************************************************/

using System;

namespace CSMultiThreadedWebDownloader
{
    public class ErrorEventArgs:EventArgs
    {
        public Exception Error { get; set; }
    }
}
